xmonad-contrib.cabal

```diff
@@ -383,31 +385,41 @@ test-suite tests
                   Selective
                   SwapWorkspaces
                   Utils
-                  ExtensibleConf
                   XMonad.Actions.CycleRecentWS
                   XMonad.Actions.CycleWS
                   XMonad.Actions.FocusNth
+                  XMonad.Actions.GridSelect
                   XMonad.Actions.PhysicalScreens
                   XMonad.Actions.RotateSome
                   XMonad.Actions.SwapWorkspaces
                   XMonad.Actions.TagWindows
+                  XMonad.Actions.WindowBringer
                   XMonad.Hooks.ManageDocks
+                  XMonad.Hooks.ManageHelpers
+                  XMonad.Hooks.UrgencyHook
```

Comment:
Any reason all these modules were added?

Reply:
That's to silence the stack warnings - I simply copied the list it suggested.

Reply:
Hmmm, I can't see the warnings in the CI though. I'm not familiar with the build though, so maybe wait for someone else with more experience to come along

Reply:
Yes, there are no warnings in the CI - because stack pulls the older version of `random` package (1.1 specifically).
I think this happens because stack is configured to use LTS Haskell 17.14 (`resolver: lts-17.14` line in stack-master.yaml), and this LTS includes random version 1.1, not 1.2.

Reply:
Nevermind, I understand now